docs/workflow/building/libraries/cross-building.md

```diff
@@ -87,6 +87,10 @@ You can also build just managed code with:
 
```

Comment:
`./build.sh --arch arm /p:BuildNative=false`

This command does not build just managed code. It builds the runtime unmanaged code too.